### PR TITLE
Adding txn status check to RPC module

### PR DESCRIPTION
**Pull request by tifrel**

### CODE REVIEW

File: packages/rpc/src/methods/txnstatus.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    id: 'dontcare',
+    method: 'tx',
+    params: [txnHash, senderId],
+  });
+  if (res.error) throw res.error;
+
+  let failure = false;
+  let pending = false;
+  res.result.receipts_outcome.forEach((outcome: any) => {
+    console.log(outcome);
```
Comment (cif) [inline packages/rpc/src/methods/txnstatus.ts:20]: 🧹

File: packages/rpc/src/util.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { NEAR_RPC_URL } from "./constants";
+import superagent from "superagent";
+
+export const requestFromNearRpc = async (
+  body: Record<string, any>
+): Promise<Record<string, any>> => {
+  const res = await superagent
```
Comment (rubenmarcus) [inline packages/rpc/src/util.ts:7]: is superagent really needed ? I mean pure fetch wont work?

File: packages/rpc/src/util.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    .post(NEAR_RPC_URL)
+    .send(body)
+    .set("Content-type", "application/json");
+
+  return res.body;
```
Comment (rubenmarcus) [inline packages/rpc/src/util.ts:12]: and if res fail?

File: packages/rpc/src/methods/blockheight.ts [hunk 4]
```diff
@@ -1,15 +1,11 @@
...
-    })
-    .set('Content-type', 'application/json');
-  return res.body.result.sync_info.latest_block_height;
+  const res = await requestFromNearRpc({
+    jsonrpc: "2.0",
+    id: "dontcare",
+    method: "status",
+    params: [],
+  });
+  return res.result.sync_info.latest_block_height;
```
Comment (rubenmarcus) [inline packages/rpc/src/methods/blockheight.ts:10]: and if res is empty , null or undefined?

File: packages/rpc/src/methods/txnstatus.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    jsonrpc: "2.0",
+    id: "dontcare",
+    method: "tx",
+    params: [txnHash, senderId],
+  });
+  if (res.error) throw res.error;
+
+  let failure = false;
+  let pending = false;
+  res.result.receipts_outcome.forEach((outcome: any) => {
```
Comment (rubenmarcus) [inline packages/rpc/src/methods/txnstatus.ts:19]: is there a situation where result  is undefined and result.receipts_outcome are not an array?

**Review by rubenmarcus — CHANGES_REQUESTED** [review]
just some questions before approve it, but it seems ok

File: packages/rpc/src/util.ts (hunk 2 above)
Comment (tifrel) [reply packages/rpc/src/util.ts:7]: Pure fetch won't work inside NodeJS

**Review by rubenmarcus — APPROVED** [review]
LGTM, just fix the conflicts

**Review by sainthiago — APPROVED** [review]

**Pull request merged**